### ISSUE TITLE
Recommendation: NAV for groupings of navigation links along with an accessible name

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#1362]
A general accessibility recommendation is to use the landmark NAV for groupings of navigation links. This could be done in the header and footer.

Also each NAV list should have an accessible name. The form-builder left and top NAVs use an arial-label correctly. Though a general accessibility recommendation is to have NAV label visible for sited users also, unless it is really obvious like a NAV element in a HEADER or FOOTER.

![Screenshot 2022-11-29 at 4.21.03 PM.png](https://images.zenhubusercontent.com/63321fcd2e391c72611f2316/38a4611c-0506-4bc4-911f-fd9a9d31664c)

![Screenshot 2022-11-29 at 4.22.48 PM.png](https://images.zenhubusercontent.com/63321fcd2e391c72611f2316/8a667e78-60ce-4812-8508-9fc2322420ab)
